README.md

```diff
@@ -84,6 +86,17 @@ which is used for local run/debug, because the integration tests will remove all
               Respective issues are created: 
                 - [Payment/Order#applicationContext property is not available](https://github.com/paypal/PayPal-Java-SDK/issues/330#issuecomment-356008914)
                 - [Payment application_context is not documented](https://github.com/paypal/PayPal-REST-API-issues/issues/179)
+                
+            - `description` (custom field): if specified - [`Payment#transaction#description`](https://developer.paypal.com/docs/api/payments/#definition-transaction)
+            field is set to this value. Otherwise the description fallback to the string:
```

Comment:
`Otherwise the description falls back to the following string: "Reference: ${payment.custom.fields.reference}"` and you might skip below `where` statement.